### DISCUSSION TITLE
How to use `is_interpolation` on nested config?

### DISCUSSION THREAD

**Discussion by merelcht** [omry/omegaconf#1074] [Q&A] [answered]
With a configuration like this:
```python
cfg = OmegaConf.create({
   "foo" : 10,
    "bar": {
          "bla":"${foo}"
      }
   })
```

How can I check if my key "bla" has an interpolation value or not? I've tried all of the following:

-  `OmegaConf.is_interpolation(cfg, "bar.bla")`
-  `OmegaConf.is_interpolation(cfg, "bar:bla")`
- `OmegaConf.is_interpolation(cfg, "bar[bla]")`

But all of those return "False", where I'm expecting to get "True".

**Comment by omry** [comment, marked as answer]
You need to operate on the parent Node.
Something like:
```python
OmegaConf.is_interpolation(cfg.bar, "bla")
```

If you want to drill in programmatically , you can use OmegaConf.select to select the parent node first:
e.g, to check if foo.bar.baz.biz is an interpolation:
```python
node = OmegaConf.select(cfg, "foo.bar.baz")
OmegaConf.is_interpolation(node, "biz")
```

**Reply by merelcht** [reply]
Thanks for the quick response @omry !

**Comment by omry** [comment]
Happy to help.
Feel free to ping me about Kedro and Hydra.
While I am not very actively involved I still care about it greatly and would love to see it get adopted by more frameworks.
